src/uct/ib/base/ib_iface.c

```diff
@@ -553,15 +587,16 @@ void uct_ib_iface_fill_ah_attr_from_gid_lid(uct_ib_iface_t *iface, uint16_t lid,
 void uct_ib_iface_fill_ah_attr_from_addr(uct_ib_iface_t *iface,
                                          const uct_ib_address_t *ib_addr,
                                          unsigned path_index,
-                                         struct ibv_ah_attr *ah_attr)
+                                         struct ibv_ah_attr *ah_attr,
+                                         enum ibv_mtu *path_mtu)
 {
     union ibv_gid  gid;
     uint16_t       lid;
 
     ucs_assert(!uct_ib_iface_is_roce(iface) ==
                !(ib_addr->flags & UCT_IB_ADDRESS_FLAG_LINK_LAYER_ETH));
 
-    uct_ib_address_unpack(ib_addr, &lid, &gid);
+    uct_ib_address_unpack(ib_addr, &lid, &gid, path_mtu);
```

Comment:
here we can check if path_mtu==0 set it to iface->mtu

Reply:
we can't do this here because `path_mtu` is present only in RC iface config.

Reply:
can we move it to base? it could be relevant for UD as well

Reply:
done